MVCapplication/src/main/resources/ValidationMessages_ru.properties

```diff
@@ -0,0 +1,17 @@
+error.email.notnull=\u0412\u0432\u0435\u0434\u0438\u0442\u0435 e-mail
+error.email.size=\u0414\u043B\u0438\u043D\u0430 \u0438\u043C\u0435\u0439\u043B\u0430 \u0434\u043E\u043B\u0436\u043D\u0430 \u0431\u044B\u0442\u044C \u043C\u0435\u0436\u0434\u0443 {min} \u0438 {max} \u0441\u0438\u043C\u0432\u043E\u043B\u043E\u0432
+error.email.exists=\u0422\u0430\u043A\u043E\u0439 \u0438-\u043C\u0435\u0439\u043B \u0443\u0436\u0435 \u0437\u0430\u0440\u0435\u0433\u0438\u0441\u0442\u0440\u0438\u0440\u043E\u0432\u0430\u043D \u0432 \u0441\u0438\u0441\u0442\u0435\u043C\u0435!
+error.password.notnull=\u0412\u0432\u0435\u0434\u0438\u0442\u0435 \u043F\u0430\u0440\u043E\u043B\u044C
+error.password.size=\u041F\u0430\u0440\u043E\u043B\u044C \u0434\u043E\u043B\u0436\u0435\u043D \u0431\u044B\u0442\u044C \u043E\u0442 {min} \u0434\u043E {max} \u0441\u0438\u043C\u0432\u043E\u043B\u043E\u0432
+error.name.notnull=\u0412\u0432\u0435\u0434\u0438\u0442\u0435 \u0438\u043C\u044F
+error.name.size=\u0418\u043C\u044F \u0434\u043E\u043B\u0436\u043D\u043E \u0431\u044B\u0442\u044C \u043E\u0442 {min} \u0434\u043E {max} \u0441\u0438\u043C\u0432\u043E\u043B\u043E\u0432
+error.role.notnull=\u0412\u044B\u0431\u0435\u0440\u0438\u0442\u0435 \u0442\u0438\u043F \u0430\u043A\u043A\u0430\u0443\u043D\u0442\u0430
+error.address.size=\u0410\u0434\u0440\u0435\u0441\u0430 \u0434\u043E\u043B\u0436\u043D\u0430 \u0431\u044B\u0442\u044C \u043E\u0442 {min} \u0434\u043E {max} \u0441\u0438\u043C\u0432\u043E\u043B\u043E\u0432
+error.productName.notblank=\u0412\u0432\u0435\u0434\u0438\u0442\u0435 \u043D\u0430\u0437\u0432\u0430\u043D\u0438\u0435 \u043F\u0440\u043E\u0434\u0443\u043A\u0442\u0430
```

Comment:
As said by https://en.wikipedia.org/wiki/.properties : The encoding of a .properties file is ISO-8859-1, also known as Latin-1. All non-Latin-1 characters must be entered by using Unicode escape characters, e.g. \uHHHH where HHHH is a hexadecimal index of the character in the Unicode character set.